cves/kernel/CVE-2014-3183.yml

```diff
@@ -56,6 +67,9 @@ description_instructions: |
   Your target audience is people just like you before you took any course in
   security
 description:
+  Heap overflow in handling of HID reports sent to Logitech Unifying receivers
```

Comment:
Definitely try to dum this down. For example, I'm not sure what HID reports are. If you feel a term is essential to the description of the vulnerability, make sure to give a quick explanation of what it is.

Reply:
I expanded more on the description.